index.html

```diff
@@ -58,5 +64,185 @@ <h3 class="dev-intro">I'am a Software Developer</h3>
         </li>
       </ul>
     </section>
+
+    <!-- WORK SECTION -->
+
+    <section class="main-wrap">
+      <article class="work-section">
+        <div class="work-title">
+          <h2>My Recent Works</h2>
+          <div class="rule"></div>
+          <div class="recent-rule"></div>
+        </div>
+        <div class="main-article">
+          <div class="article-img">
+            <img class="hide-desktop" src="image/Tonic.png" alt="Tonic Image" />
+            <!-- <img class="hide-mobile" src="image/tonic-desktop.png" alt="" /> -->
+          </div>
+          <div class="article-text">
+            <div class="multi-post">
+              <h3>Multi-Post Stories</h3>
+              <p>
+                A daily selection of privately personalized reads; no account or
+                sigh-ups required has been the industry's standard dummy text
+                ever since the 1500s, when unkown printer took a standard dummy
+                text.
+              </p>
+            </div>
+            <div class="work-languages">
+              <ul class="work-list">
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] Please the button "See project" follows the button styling: Active, Hover and Pressed 
- [ ] see design Link: [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)


![ok](https://user-images.githubusercontent.com/63972670/164675556-54591159-5629-4e9a-a1cf-ec636af423e2.PNG)